app/services/articles/feeds/weighted_query_strategy.rb

```diff
@@ -460,10 +460,10 @@ def final_order_logic(articles)
         when "final_order_by_random_weighted_to_comment_score"
           articles.order(Arel.sql("RANDOM() ^ (1.0 / greatest(articles.comment_score, 0.1)) DESC"))
         when "final_order_by_random_weighted_to_last_comment_at"
+          # rubocop:disable Layout/LineLength
           articles
-            .order(
-              Arel.sql("RANDOM() ^ (1.0 / (1 + (extract(epoch from now() - articles.last_comment_at)::integer)) ASC"),
-            )
+            .order(Arel.sql("RANDOM() ^ (1.0 / (min(1, (extract(epoch from now() - articles.last_comment_at)::integer)) ASC"))
```

Comment:
I _think_ I set you up for a mistake - this should be the larger of the (likely nonzero and likely positive) difference and 1, not the smaller?

 ```sql
(1.0 / max(1, (extract(epoch from now() - articles.last_comment_at)::integer))
```

Reply:
I agree with Dan here. I think it will always be resolve to `1`